Script/item.dat

```diff
@@ -902,7 +1015,7 @@ pickaxe /* meleeweapon-> */
   BitmapPos = 0, 96;
   FormModifier = 35;
   NameSingular = "pick-axe";
-  Alias == "pick-ax";
+  Alias = { 3, "pick-ax", "pickaxe", "pick axe"; }
```

Comment:
Cool, didn't know this was possible 👍

Reply:
Me neither, we should make more extensive use of this to get around some of the weirder text-adventure-game nonsense you have to do to wish for certain items.

e.g. Do you want some boots of kicking? Sure, but don't use "boots" otherwise I have no idea what you mean. Tell me "boot of kicking" and I'll give you two!

Reply:
Yes, I tried to cover the most likely wishes (immortality) and spellings people could use. ;) If you have any wishes that are missing, tell me.